index.js

```diff
@@ -4,23 +4,58 @@
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function addNewStore(stores, store) {}
+function addNewStore(stores, store) {
+  const result = [];
+  {
+    stores.push(store)
+
+
+
+  }
+  return stores
+}
+//Brainstorming Attempts---V
+//console.log(stores)
+//let newStores = stores(i) ;
+//for (let i = 0; i < stores.length - 1; i++)
+//Array.prototype.push()
+//push() the new store to the end of the array in the object
 
 /**
  * Removes a store object at the given position.
  * @param {Object[]]} stores - An array of store objects.
  * @param {number} index - A number representing the index of the store to be removed from the array.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function removeStoreAtPosition(stores, index) {}
+function removeStoreAtPosition(stores, index) {
+  const result = [];
+  {
```

Comment:
It looks like there is some misunderstanding with the appropriate syntax to use when and where. Sign up to office hours if you are still unsure.